report_thesis/src/sections/results/results.tex

```diff
@@ -1,11 +1,5 @@
 \section{Results}
-Analysis / Experimental / Empirical Evaluation (the analysis, comparing our method to SoA or a baseline, ablation study)
-\begin{itemize}
-    \item System used (for repeatability)
-    \item Datasets / people used
-    \item Metrics used
-    \item Results
-    \item Discussion
-\end{itemize}
+We present the results of our experiments in this section.
 
-\input{sections/results/visual_analysis.tex}
\ No newline at end of file
+\input{sections/results/initial_results.tex}
+\input{sections/results/visual_analysis.tex}
```

Comment:
Probably makes more sense to showcase the visual analysis to prove our data partitioning method works before we present our results.

```suggestion
\input{sections/results/visual_analysis.tex}
\input{sections/results/initial_results.tex}
```